src/views/download-file.njk

```diff
@@ -0,0 +1,22 @@
+{% extends 'layout.njk' %}
+
+{% block pageTitle %}
+  {% if errorList.length > 0 %}
+    Error:
+  {% endif %}
+  Sign in
```

Comment:
This is quite surprising especially that user will be signed in already at this point. Is that what prototype says? If so get in touch with UX to raise question whether it is correct.